container/docker/handler.go

```diff
@@ -248,6 +251,7 @@ func newDockerContainerHandler(
 	handler.image = ctnr.Config.Image
 	handler.networkMode = ctnr.HostConfig.NetworkMode
 	handler.deviceID = ctnr.GraphDriver.Data["DeviceId"]
+	handler.restartCount = ctnr.RestartCount
```

Comment:
Yes, actually the new handler defines how the info is going to be gathered by GetStats function. So I assume this point of review is solved :)